src/requirements.txt

```diff
@@ -2,26 +2,11 @@
 # pandas
 # sklearn
 # clustergrammer_widget
-beautifulsoup4==4.12.3
-black==24.1.1
-coverage==7.4.1
-cryptography==42.0.2
-decorator==5.1.1
-flake8==7.0.0
-html5lib==1.1
-idna==3.6
-isort==5.13.2
+ipython==8.10.0
```

Comment:
A much smaller list! Those removed aren't directly referenced by KBase code, and I figure that the Jupyter requirements can handle their own needs.